app/commands/webhooks/process_workflow_run_update.rb

```diff
@@ -4,7 +4,20 @@ class Webhooks::ProcessWorkflowRunUpdate
   initialize_with params: Mandate::KWARGS
 
   def call
-    return unless params[:action] == 'completed'
-    return unless params[:conclusion] == 'success'
+    Tooling::HandleRepresenterDeploy.defer(track) if handle_representer_deploy?
   end
+
+  private
+  def handle_representer_deploy?
+    params[:action] == 'completed' &&
+      params[:conclusion] == 'success' &&
+      params[:head_branch] == 'main' &&
+      params[:path] == '.github/workflows/deploy.yml' &&
```

Comment:
This is safe to do as the `deploy.yml` file is automatically synced to all repos via the `org-wide-files` repo and is thus guaranteed to have the same name across repos